### PR TITLE
Update to Bevy 0.15

### LINKED ISSUE #96: Prepare for Bevy 0.15

### PR DESCRIPTION
**Pull request by CarlosEduR**
Closes #96

- [x] update Bevy to 0.15
- [x] test examples
- [ ] TBD

### CODE REVIEW

**Comment by CarlosEduR** [comment]
almost there, spatial example needs some update yet

cc @GitGhillie

File: examples/audio_control.rs [hunk 1]
```diff
@@ -65,10 +65,11 @@ fn audio_control(query: Query<&AudioSource>, input: Res<ButtonInput<KeyCode>>) {
...
-        TextSection::from("P: Play \n"),
-        TextSection::from("T: Toggle \n"),
-    ]));
+    let text_entity = commands.spawn(Text::default()).id();
+    commands.entity(text_entity).with_children(|parent| {
+        parent.spawn(TextSpan::new("Controls: \n"));
+        parent.spawn(TextSpan::new("S: Stop \n"));
+        parent.spawn(TextSpan::new("P: Play \n"));
+        parent.spawn(TextSpan::new("T: Toggle \n"));
+    });
```
Comment (GitGhillie) [inline examples/audio_control.rs:74]: Small tip:
```suggestion
    commands.spawn(Text::default()).with_children(|parent| {
        parent.spawn(TextSpan::new("Controls: \n"));
        parent.spawn(TextSpan::new("S: Stop \n"));
        parent.spawn(TextSpan::new("P: Play \n"));
        parent.spawn(TextSpan::new("T: Toggle \n"));
    });
```

File: examples/parameters.rs [hunk 2]
```diff
@@ -128,11 +128,13 @@ fn set_hour(
...
-    commands.spawn(TextBundle::from_sections([
-        TextSection::from("Controls: \n"),
-        TextSection::from("Arrow Up: Increase Rain \n"),
-        TextSection::from("Arrow Down: Decrease Rain \n"),
-        TextSection::from("E: Change time to Evening \n"),
-        TextSection::from("M: Change time to Morning \n"),
-    ]));
+    let text_entity = commands.spawn(Text::default()).id();
+    commands.entity(text_entity).with_children(|parent| {
+        parent.spawn(TextSpan::new("Controls: \n"));
```
Comment (GitGhillie) [inline examples/parameters.rs:133]: Same here

**Review by GitGhillie — COMMENTED** [review]
Going in the right direction :+1:

File: examples/audio_control.rs (hunk 1 above)
Comment (CarlosEduR) [reply examples/audio_control.rs]: nice, code updated

**Comment by CarlosEduR** [comment]
all examples updated and working

**Review by GitGhillie — DISMISSED** [review]
Thanks! Looks good to me

**Comment by Salzian** [comment]
Thank you for your contribution!

**Pull request merged**